microsoft-edge/webview2/concepts/versioning.md

```diff
@@ -3,89 +3,152 @@ description: Version Models used for Microsoft Edge WebView2
 
-If WebView2 Runtime updates are stopped, ensure you understand the minimum version of the [WebView2 Runtime][MicrosoftDeveloperEdgeWebview2] required by your app.  Consider the following two items:  
+## Matching the Runtime version with the SDK version
 
-1.  The minimum required version of the SDK in order to successfully load a webview2 instance is found in the WebView2 [Release Notes][Webview2ReleaseNotes] under **Minimum Microsoft Edge version to load**.  The minimum version to load required by the SDK only changes when a breaking change occurs in the web platform.  For example, for SDK version [1.0.622.22][Webview2ReleaseNotes1062222], you must install either the [WebView2 Runtime][MicrosoftDeveloperEdgeWebview2] or a [non-stable Microsoft Edge channel][MicrosoftedgeinsiderDownload] with a build number of `86.0.616.0` or newer.   
-1.  The minimum required version of the NuGet package required to support the interfaces and APIs in your app is found in the WebView2 [Release Notes][Webview2ReleaseNotes] under **Full API Compatibility**.  New interfaces and APIs are added periodically to WebView2.  APIs and interfaces bundled in an SDK require different versions of the WebView2 Runtime, because APIs and interface are added to the SDK at different times.  The required WebView2 Runtime version matches the build number, the third number, of the SDK version the API was first introduced in.  For example, a new API or interface added in SDK version [1.0.622.22][Webview2ReleaseNotes1062222] requires the WebView2 Runtime version `86.0.622.0` or newer.  An API or interface added in a later SDK release requires a WebView2 Runtime that has the same version number as the SDK.  To help you determine if the WebView2 Runtime version supports an interface or API, navigate to [Determine WebView2 Runtime requirement](#determine-webview2-runtime-requirement).  
-    
+A WebView2 production app requires that the [WebView2 Runtime][MicrosoftDeveloperEdgeWebview2] is present on the client.  In the Evergreen distribution approach, the installed WebView2 Runtime automatically updates to the latest version available; however, a user or IT admin might choose to prevent automatic updating of the WebView2 Runtime.  The resulting outdated Runtime on the client might cause compatibility issues between your updated WebView2 app that uses new APIs from a recent SDK, and the older Runtime.
+
+In case updating the WebView2 Runtime is prevented on the client, make sure that you know the minimum version number of the [WebView2 Runtime][MicrosoftDeveloperEdgeWebview2] that is required by your app.  The minimum required Runtime version to support the General Availability release of the SDK (build 616) is older than for the latest Runtime.  The latest Runtime supports all APIs that are in the latest SDK release build.
+
+### Minimum version for GA API support
```

Comment:
For  `### Minimum version for GA API support` and `### Minimum version for full API support`, didn't we talk about the same things in similar details in the first few paragraphs of the doc?